_partials/_migrate_post_schema_caggs_etal.md

```diff
@@ -0,0 +1,165 @@
+## Migrate schema post-data
+
+When you have migrated your table and hypertable data, migrate your PostgreSQL
+schema post-data. This includes information about constraints.
+
+<Procedure>
+
+### Migrating schema post-data
+
+1.  At the command prompt, dump the schema post-data from your source database
+    into a `dump_post_data.bak` file, using your source database connection details. Exclude
+    Timescale-specific schemas. If you are prompted for a password, use your
+    source database credentials:
+
+    ```bash
+    pg_dump -U <SOURCE_DB_USERNAME> -W \
+    -h <SOURCE_DB_HOST> -p <SOURCE_DB_PORT> -Fc -v \
+    --section=post-data --exclude-schema="_timescaledb*" \
+    -f dump_post_data.bak <DATABASE_NAME>
+    ```
+
+1.  Restore the dumped schema post-data from the `dump_post_data.bak` file into
+    your Timescale database, using your connection details. To avoid permissions
+    errors, include the `--no-owner` flag:
+
+    ```bash
+    pg_restore -U tsdbadmin -W \
+    -h <HOST> -p <PORT> --no-owner -Fc \
```

Comment:
Do you need the `-Fc` here? I think `pg_restore` figures out this itself and the documentation for [`pg_dump` and `pg_restore`](https://www.postgresql.org/docs/current/app-pgdump.html) does not use it.